cs/src/Management/TunnelRequestOptions.cs

```diff
@@ -113,6 +113,11 @@ public class TunnelRequestOptions
         /// </summary>
         public bool ForceRename { get; set; }
 
+        /// <summary>
+        /// Limits the number of tunnels returned when searching or listing tunnels.
```

Comment:
Are we going to have a default limit if the request option is not provided?
Also, are we going to limit the `Limit` itself?

Reply:
If limit is not provided then all results will be returned, if the limit is larger than the number of results then it will just return all results.